routes/items.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
```

Comment:
You aren't attaching this router to the main express app at any point.

Reply:
unsure what you are saying

Reply:
Have you been able to use postman to check that your /items endpoint works?